src/service/url-replacements-impl.js

```diff
@@ -462,6 +462,13 @@ export class GlobalVariableSource extends VariableSource {
       });
     });
 
+    // Returns the incremental engaged time since the last push.
```

Comment:
The way you implemented is not the incremental from last push (ping), but the last read of the var, because you reset the var after reading the var.

Things can become problematic if multiple analytics vendors on the page are reading this value. Since the number is a singleton, so each read will interfere each other and no one will get the correct number.